### PR TITLE
Fix bckg estimate method

### PR DESCRIPTION
**Pull request by dapparu**
- Fix cutIndex way
- Fix pseudo experiments dependance
- Fix error bars

### CODE REVIEW

File: Analyzer/test/Tamas/submitCrabJobsData.py [hunk 1]
```diff
@@ -45,12 +45,12 @@
 config.Data.unitsPerJob = 50
 config.Data.publication = True
 config.Data.outputDatasetTag = config.General.requestName
-config.Data.outLFNDirBase = '/store/user/tvami/HSCP'
+config.Data.outLFNDirBase = '/store/user/dapparu/HSCP/Analysis/CMSSW_10_6_27'
```
Comment (tvami) [inline Analyzer/test/Tamas/submitCrabJobsData.py:48]: hi @dapparu I think we should have a `/test/Dylan/` folder and have these files there. Under Tamas these configs should have my settings. Can you please do that?

**Comment by tvami** [comment]
It's a bit unclear to me what was wrong the the nPE, now you added the check for it's size, but how does that solve the nPE = 100, 200, 500 problems? Can you please elaborate a bit on that? thanks

File: Analyzer/test/Tamas/submitCrabJobsData.py (hunk 1 above)
Comment (dapparu) [reply Analyzer/test/Tamas/submitCrabJobsData.py:48]: I'll do that sure. Sorry.

**Comment by dapparu** [comment]
> It's a bit unclear to me what was wrong the the nPE, now you added the check for it's size, but how does that solve the nPE = 100, 200, 500 problems? Can you please elaborate a bit on that? thanks

The issue came from histograms gave as reference and they were modified during each pseudo-experiment then it implied a kind of convergence which is absolutely not what we want. I think there is still a little work which could be done to save computation time.

**Pull request merged**